baiji/test_s3.py

```diff
@@ -26,9 +26,8 @@ def setUp(self):
         self.s3_path = loc.path
 
         b = s3.S3Connection()._bucket(self.bucket) # pylint: disable=protected-access
-        for key in b.list(self.s3_path[1:]):
-            b.delete_key(key)
-        self.assertEqual(len([x for x in b.list(self.s3_path[1:])]), 0) # just make sure we're starting with a clean slate
+        b.delete_key(self.s3_path[1:] + '.tempdir')
```

Comment:
Yeah, that file is written by `s3.path.gettmpdir()` and is guaranteed to be the only file in the directory. The old "iterate and delete all" code dated to a version of this that was sharing the temp dir amongst all the tests.